src/App.js

```diff
@@ -9,14 +8,40 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      movie: null
+      movies: [],
+      // movies: movieData.movies,
+      movie: null,
+      error: ''
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => {
+        // console.log(response)
+        if(!response.ok) {
+          throw new Error("Oopsies! Something went wrong 🤡")
+        } else {
+          return response.json()
+        }
+      })
+      // .then(data => this.setState({ ideas: data }))
+      .then(movies => this.setState({ movies: movies.movies }))
+      .catch(error => {
+        this.setState({ error: "ERROR: " + error.message })
+        // this.setState({ error: 'An error occured. Please try again!' })
+      })
+  }
+
   findMovie = (id) => {
-    const foundMovie = this.state.movies.find(movie => movie.id === id);
-    this.setState({movie: foundMovie});
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+      .then(response => response.json())
```

Comment:
Nice work here as well.  Remember to include your conditional logic for checking if the `response.ok` is good!